samples/apply-stretch-renderer/src/main/java/com/esri/arcgismaps/sample/applystretchrenderer/screens/ApplyStretchRendererScreen.kt

```diff
@@ -0,0 +1,291 @@
+                            )
+                        }
+
+                        StretchType.StandardDeviation -> {
+                            StdDevSettings(
+                                stdDevFactor = stdDevFactor,
+                                onStdDevFactorChange = mapViewModel::updateStdDeviationFactor
+                            )
+                        }
+                    }
+
+                    Row(
+                        modifier = Modifier.fillMaxWidth(),
+                        horizontalArrangement = Arrangement.SpaceEvenly
+                    ) {
+                        OutlinedButton(
+                            onClick = {
+                                isBottomSheetVisible = false
+                                mapViewModel.dismissChanges()
+                            }
+                        ) {
+                            Text("Dismiss")
+                        }
+                        Button(
+                            onClick = {
+                                mapViewModel.updateRenderer()
+                                isBottomSheetVisible = false
+                            }
+                        ) {
+                            Text("Update Renderer")
+                        }
+                    }
+                }
+            }
+
+            // Error dialog
+            mapViewModel.messageDialogVM.apply {
+                if (dialogStatus) {
+                    MessageDialog(
+                        title = messageTitle,
+                        description = messageDescription,
+                        onDismissRequest = ::dismissDialog
+                    )
+                }
+            }
+        },
+        floatingActionButton = {
+            if (!isBottomSheetVisible) {
+                FloatingActionButton(
+                    modifier = Modifier.padding(bottom = 36.dp, end = 12.dp),
+                    onClick = { isBottomSheetVisible = true }
+                ) {
+                    Icon(
+                        Icons.Filled.Settings,
+                        contentDescription = "Show Renderer Settings"
+                    )
+                }
+            }
+        }
+    )
+}
+
+// UI for Min-Max stretch parameters
+@Composable
+fun MinMaxSettings(
+    minValue: Double,
+    maxValue: Double,
+    onMinValueChange: (Double) -> Unit,
+    onMaxValueChange: (Double) -> Unit,
+    valueRange: ClosedFloatingPointRange<Float> = 0f..255f
+) {
+    var range by remember { mutableStateOf(minValue.toFloat()..maxValue.toFloat()) }
+
+    Column(horizontalAlignment = Alignment.CenterHorizontally) {
+        Text("Min-Max Parameters", style = MaterialTheme.typography.titleMedium)
+
+        Text(text = "Min Value: ${range.start.toInt()}  Max Value: ${range.endInclusive.toInt()}")
+
+        RangeSlider(
+            value = range,
+            onValueChange = { newRange ->
+                if ( newRange.start < newRange.endInclusive ) {
```

Comment:
Nitpick: Formatting, 

```suggestion
                if (newRange.start < newRange.endInclusive) {
```